zerver/tests/test_auth_backends.py

```diff
@@ -384,38 +384,15 @@ def test_social_auth_backends(self) -> None:
         backends_to_test = {
             'google': {
                 'urls': [
-                    {
-                        'url': "https://accounts.google.com/o/oauth2/token",
-                        'method': httpretty.POST,
-                        'status': 200,
-                        'body': json.dumps(token_data_dict),
-                    },
-                    {
-                        'url': "https://www.googleapis.com/oauth2/v3/userinfo",
-                        'method': httpretty.GET,
-                        'status': 200,
-                        'body': json.dumps(google_email_data),
-                    },
+                    # The limited process that we test here doesn't require mocking any urls.
                 ],
                 'backend': GoogleAuthBackend,
             },
             'github': {
                 'urls': [
-                    {
-                        'url': "https://github.com/login/oauth/access_token",
-                        'method': httpretty.POST,
-                        'status': 200,
-                        'body': json.dumps(token_data_dict),
-                    },
-                    {
-                        'url': "https://api.github.com/user",
-                        'method': httpretty.GET,
-                        'status': 200,
-                        'body': json.dumps(dict(email=user.email, name=user.full_name)),
-                    },
```

Comment:
This test had a bunch of URLs it mocked that weren't involved in the limit process tested here. This was detected thanks to ``responses.RequestsMock(assert_all_requests_are_fired=True)`` verifying that all the registered URLs are actually used.